docs/TBG_macros.cfg

```diff
@@ -90,6 +90,8 @@ TBG_periodic="--periodic 1 0 1"
 # Enables moving window (sliding) in your simulation
 TBG_movingWindow="-m"
 
+# stop the moving window after given simulation step
+TBG_stopWindow="--stopWindow 1337"
 ################################################################################
```

Comment:
pls give me an empty line :)

Reply:
You and your white space fetish^^ ( I will add the empty line)

Reply:
The - symmetry - is - broken! xD

Reply:
I forgot to push my changes